src/ui/ConfigEditor.tsx

```diff
@@ -123,13 +123,17 @@ export class ConfigEditor extends PureComponent<Props, State> {
               placeholder="Select your checkmk edition"
             />
           </InlineField>
-          <InlineField label="Backend" labelWidth={12} tooltip="Choose REST-API for checkmk >= 2.2">
+          <InlineField
+            label="Backend"
```

Comment:
```suggestion
            label="Version"
```
open to discuss this, i think backend and version do not match. if you change this, you would have to rename backend in the error message above.

Reply:
i would go with version, as the dropdown let's you choose a version and not a backend. changing it to a version dropdown was a requirement/idea from lars.

Reply:
Sorry, forgot to add the chance to my commit. Should be fixed now.